mtest/libmscore/text/tst_text.cpp

```diff
@@ -763,6 +765,48 @@ void TestText::testDropUnicodeAfterSMUFLwhenCursorSetToSymbol()
       QCOMPARE(text->xmlText(), QString("<sym>noteheadWhole</sym>𝄎"));
       }
 
+//---------------------------------------------------------
+///   testDropBasicUnicodeWhenNotInEditMode
+///     Simple test dropping basic unicode, but excercising the path when edit mode not already engaged
+//---------------------------------------------------------
+
+void TestText::testDropBasicUnicodeWhenNotInEditMode()
+      {
+      Text* text = new Text(score);
+      text->initSubStyle(SubStyle::DYNAMICS);
+      text->setPlainText(QString(""));
+      text->layout();
+
+      DropData dropFSymbol;
+      FSymbol* fsymbol = new FSymbol(score);
+      fsymbol->setCode(0x4D); // Basic Unicode code for 'M'
+      dropFSymbol.element = fsymbol;
+      text->drop(dropFSymbol);
+
+      QCOMPARE(text->xmlText(), QString("M"));
+      }
```

Comment:
This is a really trivial test, but I'm putting here to be sure not break this functionality when not in edit mode.